hoodie-client/src/test/java/com/uber/hoodie/TestCleaner.java

```diff
@@ -915,7 +915,13 @@ public void testPendingCompactions(HoodieWriteConfig config, int expNumFilesDele
    * @throws IOException in case of error
    */
   private int getTotalTempFiles() throws IOException {
-    return fs.listStatus(new Path(basePath, HoodieTableMetaClient.TEMPFOLDER_NAME)).length;
+    RemoteIterator itr = fs.listFiles(new Path(basePath, HoodieTableMetaClient.TEMPFOLDER_NAME), true);
```

Comment:
Change to Iterators.size(itr) from google utils

Reply:
RemoteIterator is not inheriting from Iterator.